server/engine/src/main/java/org/eclipse/lsp/cobol/lsp/jrpc/CobolLanguageClient.java

```diff
@@ -76,9 +75,10 @@ default CompletableFuture<Void> downloadCopybooks(
   /**
    * Notifies language client that CFAST is ready
    * @param result is a result with CFAST data
+   * @return void object
    */
-  @JsonNotification("cfast/ready")
-  default void cfastReady(ExtendedApiResult result) {
+  @JsonRequest("cfast/ready")
```

Comment:
Why did you change it into a request? The language server does not expect any results and does not even attempt to wait for them.

Reply:
I change it because Notification handler cannot be async on the client side